.golangci.yaml

```diff
@@ -1,6 +1,12 @@
 linters:
   enable-all: true
 
+# all available settings of specific linters
+linters-settings:
+  govet:
+    # report about shadowed variables
+    check-shadowing: true
```

Comment:
eh, I'd like to see what it complains about.  I prefer shadowed variables to contrived names.